addons/sys_prc117f/CfgWeapons.hpp

```diff
@@ -20,6 +20,11 @@ class CfgWeapons {
         class Library {
             libTextDesc = QUOTE(NAME_PRC117F);
         };
+        
+        EGVAR(arsenalStats,frequencyMin) = 30e6;
+        EGVAR(arsenalStats,frequencyMax) = 512e6;
+        EGVAR(arsenalStats,transmitPower) = 20000;
+        EGVAR(arsenalStats,effectiveRange) = "20+km (10-20km)";
```

Comment:
I am not sure I like placing those arbitrary numbers here. It's clear it's for the Arsenal, so that's good, but can we get those from actual radio data that's already defined somehow? @Sniperhid @Soldia1138

Reply:
148 seems to validate data with ` if (_newVal > 512 || {_newVal < 30}) then {`
but 152 allows you to enter anything 
![image](https://user-images.githubusercontent.com/9376747/105246081-5d818e80-5b38-11eb-8f8e-6f66f1793385.png)
117 same
![image](https://user-images.githubusercontent.com/9376747/105246605-1d6edb80-5b39-11eb-9bcf-d7cabaad9e32.png)

could add new (non-arsenal) configs for min/max frequency for user input
but I don't know how the real radios handle this

Reply:
@IDI-Systems/acre-managers ?

Reply:
I think the way to go about this is make those not Arsenal-specific, but actually use those config entries to limit radio operation (like 148 does).